hazelcast/src/test/java/com/hazelcast/test/starter/test/HazelcastVersionLocatorTest.java

```diff
@@ -37,10 +37,11 @@
 import static com.hazelcast.test.starter.HazelcastVersionLocator.Artifact.OS_JAR;
 import static com.hazelcast.test.starter.HazelcastVersionLocator.Artifact.OS_TEST_JAR;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 @SuppressWarnings("UnstableApiUsage")
 @RunWith(HazelcastParallelClassRunner.class)
-@Category({SlowTest.class, ParallelJVMTest.class})
+@Category({ SlowTest.class, ParallelJVMTest.class })
```

Comment:
> This looks like an auto-format from your IDE - generally we avoid formatting changes (if the checkstyle passes, we're happy), so that way it does not cloud the PR's changes. Also basically all existing `@Category` annotations do not contain spaces around the curly braces.
> 
> If you haven't already, I suggest [setting up your IDE to use the Hazelcast checkstyle](https://hazelcast.atlassian.net/wiki/spaces/EN/pages/5374373/Codestyle+Code-Formatter) for local formatting.

I'm not using IntelliJ so I'm not able to set this up, but we have a mix of formatting across the codebase with regards to array initialisers having or not having spaces - the '@SuppressWarnings' are done in both ways.
I've updated my style to suit.